### DISCUSSION TITLE
Color of BAR_BORDER_PATCH

### DISCUSSION THREAD

**Discussion by mrwsl** [bakkeby/dwm-flexipatch#254] [Q&A] [answered]
How do I control the color of the border patch?

**Comment by bakkeby** [comment, marked as answer]
The bar border takes its colour from SchemeNorm. There hasn't been any explicit effort to make this further configurable. 

It is easy enough to create a new colour scheme, e.g. SchemeBar, and set that scheme here:

https://github.com/bakkeby/dwm-flexipatch/blob/c553f93b528449ed2e7d4b424f063ef3c1d3d8f6/dwm.c#L1614

**Reply by mrwsl** [reply]
Got it. Just an fyi: when you use the border and underline patch, the underline gets shifted into the number:
![image](https://user-images.githubusercontent.com/8242787/171010024-7f1a8f22-6cf2-4507-9b65-086b26d02557.png)

**Reply by bakkeby** [reply]
That is likely a bug or compatibility issue.
